Build Failure on refs/heads/main

@gemini-code-assist /jules debug this build error

**Log:**
```
ractDebugAnnotations
> Task :feature:dashboard:compileDebugKotlin
> Task :feature:dashboard:compileDebugJavaWithJavac
> Task :feature:dashboard:processDebugJavaRes
> Task :feature:dashboard:mergeDebugJavaResource
> Task :feature:dashboard:transformDebugClassesWithAsm
> Task :feature:dashboard:bundleLibCompileToJarDebug
> Task :feature:dashboard:bundleLibRuntimeToJarDebug
> Task :feature:dashboard:mergeDebugGeneratedProguardFiles
> Task :feature:dashboard:bundleLibRuntimeToDirDebug
> Task :feature:dashboard:mergeDebugConsumerProguardFiles
> Task :feature:dashboard:syncDebugLibJars FROM-CACHE
> Task :feature:dashboard:bundleDebugAar
> Task :feature:dashboard:assembleDebug
> Task :feature:ar:compileDebugKotlin

> Task :feature:editor:compileDebugKotlin
w: ATTENTION!
This build uses unsafe internal compiler arguments:

-XXLanguage:+PropertyParamAnnotationDefaultTargetMode

This mode is not recommended for production use,
as no stability/compatibility guarantees are given on
compiler or generated code. Use it at your own risk!


> Task :feature:ar:compileDebugJavaWithJavac
> Task :feature:ar:processDebugJavaRes
> Task :feature:ar:mergeDebugJavaResource
> Task :feature:ar:transformDebugClassesWithAsm
> Task :feature:ar:bundleLibCompileToJarDebug
> Task :feature:ar:bundleLibRuntimeToJarDebug
> Task :feature:ar:mergeDebugGeneratedProguardFiles
> Task :feature:ar:mergeDebugConsumerProguardFiles
> Task :feature:ar:bundleLibRuntimeToDirDebug
> Task :feature:ar:syncDebugLibJars
> Task :feature:ar:bundleDebugAar
> Task :feature:ar:assembleDebug
> Task :feature:editor:compileDebugJavaWithJavac
> Task :feature:editor:processDebugJavaRes
> Task :feature:editor:mergeDebugJavaResource
> Task :feature:editor:transformDebugClassesWithAsm
> Task :feature:editor:bundleLibCompileToJarDebug
> Task :feature:editor:bundleLibRuntimeToJarDebug
> Task :feature:editor:mergeDebugGeneratedProguardFiles
> Task :feature:editor:mergeDebugConsumerProguardFiles
> Task :feature:editor:syncDebugLibJars FROM-CACHE
> Task :feature:editor:bundleLibRuntimeToDirDebug
> Task :feature:editor:bundleDebugAar
> Task :feature:editor:assembleDebug
> Task :app:mergeLibDexDebug
> Task :app:kspDebugKotlin

> Task :app:compileDebugKotlin
e: file:///home/runner/work/GraffitiXR/GraffitiXR/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt:480:25 No parameter with name 'tutorialModal' found.
e: file:///home/runner/work/GraffitiXR/GraffitiXR/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt:481:25 No parameter with name 'tutorialDim' found.
e: file:///home/runner/work/GraffitiXR/GraffitiXR/app/src/main/java/com/hereliesaz/graffitixr/MainScreen.kt:130:56 Unresolved reference 'planeConfirmationPending'.

> Task :app:compileDebugKotlin FAILED
gradle/actions: Writing build results to /home/runner/work/_temp/.gradle-actions/build-results/gradle-build-1776808338271.json

[Incubating] Problems report is available at: file:///home/runner/work/GraffitiXR/GraffitiXR/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.btapi.BuildToolsApiCompilationWork
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights from a Build Scan (powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.4.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1m 38s
362 actionable tasks: 216 executed, 146 from cache

```

--- a/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt
+++ b/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt
@@ -476,9 +476,7 @@ class MainActivity : ComponentActivity() {
                         helpEnabled = true,
                         helpList = activeHelpList,
                         onDismissHelp = { },
-                        tutorials = tutorials,
-                        tutorialModal = false,
-                        tutorialDim = 0.0f
+                        tutorials = tutorials
                     )
 
                     if (isRailVisible) {

--- a/app/src/main/java/com/hereliesaz/graffitixr/MainScreen.kt
+++ b/app/src/main/java/com/hereliesaz/graffitixr/MainScreen.kt
@@ -127,7 +127,7 @@ fun MainScreen(
                     }
 
                     val visibleLayers = uiState.layers.filter { it.isVisible && it.bitmap != null }
-                    val showPlaneConfirm = mainUiState.planeConfirmationPending
+                    val showPlaneConfirm = mainUiState.isInPlaneRealignment
 
                     LaunchedEffect(visibleLayers, arUiState.isAnchorEstablished, showPlaneConfirm) {
                         if (!arUiState.isAnchorEstablished || showPlaneConfirm || visibleLayers.isEmpty()) {

